pkg/interface/src/views/apps/chat/ChatResource.tsx

```diff
@@ -66,7 +66,7 @@ export function ChatResource(props: ChatResourceProps) {
   const chatInput = useRef<ChatInput>();
 
   useEffect(() => {
-    props.api.graph.getGraph(owner,name);
+    props.api.graph.getNewest(owner, name, 10);
```

Comment:
Why only the newest 10? Surely we could handle a hundred or so?

Reply:
Looks like it got increased [here](https://github.com/urbit/urbit/pull/4058/files#diff-7c7cc8a40d849e76929b674f5a1020d03a7d7742caa3eaa53b2df1564b67c799R44-R49)?

Reply:
Gotcha. I went through all the commits individually to watch it evolve.